packages/graphback-runtime/src/createRuntimeServices.ts

```diff
@@ -0,0 +1,49 @@
+import { GraphQLObjectType, GraphQLSchema } from "graphql"
+import { PubSubEngine } from 'graphql-subscriptions';
+import { GraphbackDataProvider } from './data/GraphbackDataProvider';
+import { CRUDService } from './service/CRUDService';
+import { GraphbackModelConfig } from "./GraphbackModelConfig";
+
+export type RuntimeServicesOptions<Provider extends GraphbackDataProvider, Service extends CRUDService> = {
+  schema: GraphQLSchema,
+  models: GraphbackModelConfig[]
+  db: any
+  pubSub: PubSubEngine
+  provider: new (...args: any[]) => Provider,
+  service: new (...args: any[]) => Service;
+};
+
+/**
+ * Helper function for creating array of datasources based on the model files that will 
+ * be able to connect to Postgress database using knex
+ * 
+ * @param schema 
+ * @param db 
+ * @param pubSub 
+ */
+export function createRuntimeServices<Provider extends GraphbackDataProvider, Service extends CRUDService> 
+  (servicesConfig: RuntimeServicesOptions<Provider, Service>) {
+
+  const { models, schema, db, pubSub, provider, service } = servicesConfig
+  if (!models || models.length === 0) {
+    throw new Error(`No models provided`)
+  }
+
+  return models.reduce((services: any, model: GraphbackModelConfig) => {
```

Comment:
Similar to above we do not need to have external models. this is very convoluted - we can base on schema itself